build.sh

```diff
@@ -9,5 +9,13 @@ BROWSERIFY_CMD="browserify -s detectEthereumProvider"
 mkdir -p ./dist
 rm -rf ./dist/*
 
-$BROWSERIFY_CMD src/index.js > dist/detect-provider.js
-$BROWSERIFY_CMD -p tinyify src/index.js > dist/detect-provider.min.js
+tsc --project . --outDir dist
```

Comment:
Can we move the `outDir` option to the `tsconfig` file?

Reply:
I tried but I couldn't get it to work. It should just be a top-level field, right?

Reply:
I may have been doing multiple things wrong at the time, I'll try again.

Reply:
It is a compiler option: [`outDir`](https://www.typescriptlang.org/tsconfig#outDir)

Reply:
Not top-level (🤦), but under `compilerOptions`: 02d2728